streamlit_app/app.py

```diff
@@ -31,30 +34,28 @@
 # Support(milk) = 2 / 5 = 0.4
 # Lift(milk --> butter)
 
-"""
-transaction ID     milk bread	butter	beer	diapers
-                1	1	1	0	0	0
-                2	0	0	1	0	0
-                3	0	0	0	1	1
-                4	1	1	1	0	0
-                5	0	1	0	0	0
-"""
-
-"""
-antecedents	consequents	antecedent support	consequent support	support	confidence	lift	leverage	conviction
-0	(Onion)	(Eggs)	0.6 0.8	0.6	1.00	1.25	0.12	inf
-1	(Eggs)	(Onion)	0.8	0.6	0.6	0.75	1.25	0.12	1.6
-2	(Onion, Kidney Beans)	(Eggs)	0.6	0.8	0.6	1.00	1.25	0.12	inf
-3	(Eggs, Kidney Beans)	(Onion)	0.8	0.6	0.6	0.75	1.25	0.12	1.6
-4	(Onion)	(Eggs, Kidney Beans)	0.6	0.8	0.6	1.00	1.25	0.12	inf
-5	(Eggs)	(Onion, Kidney Beans)	0.8	0.6	0.6	0.75	1.25	0.12	1.6
-"""
```

Comment:
Comments inside docstrings show up on the frontend of streamlit app. We should stick to one-line comments